nixos/modules/services/hardware/amdgpu.nix

```diff
@@ -30,10 +41,7 @@ in {
 
     hardware.opengl = lib.mkIf cfg.opencl.enable {
       enable = lib.mkDefault true;
-      extraPackages = [
-        pkgs.rocmPackages.clr
-        pkgs.rocmPackages.clr.icd
-      ];
+      extraPackages = [ pkgs.mesa.opencl ];
```

Comment:
Do we have any independent benchmarks for this claim? https://www.phoronix.com/news/Rusticl-RadeonSI-Near

Reply:
We should keep ROCm here as an option, I think, and add a global Mesa OpenCL option (that's on my list of yaks for this weekend probably). Also, which claim?

Reply:
That rusticl is faster than opencl. Or more generally speaking faster.

Reply:
I can do some quick tests to confirm faster, it's definitely better because ROCm is
- really shitty code overall (source: I've read it)
- comically large
- only officially supports RDNA3

Reply:
It's an option again.